app/controllers/concerns/foreman/controller/registration.rb

```diff
@@ -70,4 +73,21 @@ def not_found(options = nil)
 
     false
   end
+
+  def registration_url
+    uri = if params[:url].presence
```

Comment:
s/presence/present?/ to be more semantical, `presence` returns the value we don't use anywhere, `present?` returns boolean, which we're after in this condition